### ISSUE TITLE
#191 seems to be an error

### ISSUE THREAD

**Issue by elotroalex** [overview/osp#12]
It shows Heidegger Nietzsche as a title. The only book with that name might not be the one listed: 
http://www.worldcat.org/title/heidegger-nietzsche/oclc/820632288

**Comment by davidmcclure** [comment]
Yep, those matches are almost certainly false positives. This doesn't get weeded out by the validator, though, because both the title and author are infrequent words.

In a general sense, the way to improve this is to write a classifier that evaluates citation matches on a number of criteria (frequency of the tokens, presence of other citations nearby in the document, etc.) and determines if we keep or discard a text.

**Comment by elotroalex** [comment]
cool. let me know if i can help with anything
